mm2src/mm2_main/src/mm2_tests.rs

```diff
@@ -30,7 +30,7 @@ use uuid::Uuid;
 use mm2_test_helpers::for_tests::init_z_coin_native;
 
 #[cfg(all(feature = "zhtlc-native-tests", not(target_arch = "wasm32")))]
-async fn enable_z_coin(mm: &MarketMakerIt, coin: &str) -> ZcoinActivationResult {
+async fn enable_z_coin(mm: &MarketMakerIt, coin: &str) -> CoinActivationResult {
```

Comment:
I didn't run the tests with the `feature = "zhtlc-native-tests"` and missed this problem when I worked on my task. Thank you for the fix!

Reply:
Well, I was running zombie tests manually to see `is_mine` field, bcz they are ignored now as `it requires a long-running Zcoin initialization process` (blockchain has over 300k blocks).
Even with checkpoint block I had problems during testing.
Anyway we cannot manually change checkpoint block in test config every time, so we need to [dockerize](https://github.com/KomodoPlatform/atomicDEX-API/issues/927#issuecomment-1252162157) ZHTLC related tests to not ignore them. 
Just the info.